doc/manual/cookbooks/phase_diagram/doc/phase_diagram.tex

```diff
@@ -0,0 +1,83 @@
+Here, I visualize a diagram of the phase transitions implemented in the visco plastic material model, as well as a lookup table in the Steinberger material model. 
+
+\paragraph{The input file.}
+You can find the input file to run this cookbook example in \url{cookbooks/phase_diagram.prm}. For this first case, phase transitions are prescribed manually in terms of their depth, Clapeyron slope, and other key parameters. 
+
+\par The model domain is  800 km  by 800 km box. % geometry
+Initial temperature increases from 273 K on the left side to 2273 K on the right side. % initial conditions
+Pressure, on the other hand, increases from the top of the box to the bottom with a constant gradient.
+This is assured by assigning a constant density with zero expansivity.
+It differs from a classic phase diagram only in the direction of pressure increase.
+
+\par Two compositions of pyrolite and harzburgite are included in the model. % composition
+With each run, only one composition is assigned to the whole domain in order to visualize the diagrams of these two separately.
+
+\par The material model is then set up to mimic mantle phase transitions at 410, 520, and 660 km.% setup of mantle transitions
+Details of phase transitions are taken from \cite{billen2018decoupling}.
+A trick is needed to make this work: I assign the values of the density to the field of thermal capacity in the input file. % trick
+To do this, I used the phase inputs implemented in the ViscoPlastic file.
+This serves the goal of visualizing values of reference densities of phases with the 'real density' assigned with a constant value in the model.
+Inputs for this material model is listed here:
+\lstinputlisting[language=prmfile]{cookbooks/phase_diagram/doc/material_model.prm.out}
+Compared to the first example, pyrolitic phases are illustrated on a much finer scale.  % comparison 
+Meanwhile, we can still tell the phase transitions at 410, 520, and 660 km depth, respectively, marked by linear boundaries analogous to a constant Clapeyron slope.
+
+
+\paragraph{Results}
+
+\par Visualization of the model results yields a phase diagram of a pyrolitic mantle (Figure \ref{fig:phase_diagram_ph_density}). % phase diagram
+The field shown here has the reference densities of the pyrolite phases, though settings of phase transitions are over-simplified.
+One may notice that 3 transitions (i.e. one in the olivine system, two in the spinel system) are included for the 660 interfaces, % 3 transitons, higher T
+and they need to be modified at a higher temperature.
+In spite of the complexities of mantle phases, the focus of this first example is to simply illustrate this approach of visualizing it.
+Beyond showing the diagram, I have also used the 'lineout' feature in VISIT to export the data along two vertical lines at T = 1173 and T = 1673 (Figure \ref{fig:phase_diagram_ph_profile}). % linear profile
```

Comment:
Here and everywhere: use we instead of I. Also, check the quotes here as well.

Reply:
In this case, the quotes seem unnecessary, I deleted them